packages/terra-list/src/SingleSelectList.jsx

```diff
@@ -32,27 +36,17 @@ class SingleSelectList extends React.Component {
 
   static selectedIndexFromItems(items) {
     for (let i = 0; i < items.length; i += 1) {
-      if (items[i].props.isSelected === true) {
+      if (items[i].props.isSelected === true && items[i].props.isSelectable !== false) {
```

Comment:
nit: we should change this sometime. I'll log an issue for this.

Setting boolean variables to undefined is making us write some weird logic like this: 
```
items[i].props.isSelectable !== false
```
Boolean default props should always be either true of false, not undefined:
https://github.com/cerner/terra-core/blob/master/packages/terra-list/src/ListItem.jsx#L27-L30